File: SGA_Api/SGA_Api/Models/Conteos/ResultadoConteo.cs
namespace SGA_Api.Models.Conteos
{
    public class ResultadoConteo
    {
        public long OrdenId { get; set; }
        public string CodigoAlmacen { get; set; } = string.Empty;
        public string? CodigoUbicacion { get; set; }
        public string? CodigoArticulo { get; set; }
        public string? DescripcionArticulo { get; set; }
        public string? LotePartida { get; set; }
        public decimal? CantidadContada { get; set; }
        public decimal? CantidadStock { get; set; }
        public string UsuarioCodigo { get; set; } = string.Empty;
        public decimal Diferencia { get; set; }
        public string AccionFinal { get; set; } = string.Empty;
        public string? AprobadoPorCodigo { get; set; }
        public DateTime FechaEvaluacion { get; set; }
        public bool AjusteAplicado { get; set; } = false;

        // Navigation property
        public OrdenConteo Orden { get; set; } = null!;
    }
} 

File: SGA_Android/app/src/main/java/com/example/sga/data/model/conteos/ResultadoConteo.kt
package com.example.sga.data.model.conteos

data class ResultadoConteo(
    val guidID: String,
    val ordenGuid: String,
    val diferencia: Double,
    val accionFinal: String,
    val aprobadoPorCodigo: String?,
    val fechaEvaluacion: String,
    val ajusteAplicado: Boolean,
    val codigoAlmacen: String,
    val codigoUbicacion: String,
    val codigoArticulo: String,
    val descripcionArticulo: String?,
    val lotePartida: String,
    val cantidadContada: Double,
    val cantidadStock: Double,
    val usuarioCodigo: String?
)
